styles/styles.css

```diff
@@ -1029,6 +1038,11 @@ main .block.centered > div {
     font-size: var(--heading-font-size-mxl);
   }
 
+  /* section - large-padded */
+  main .section.large-padded {
+    padding: 5rem 0;
```

Comment:
These paddings should be applied to section-container level. section by itself should occupy the full space.

Reply:
Fixed